halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/SpinnakerEndpoints.java

```diff
@@ -54,5 +75,29 @@
     public String getBaseUrl() {
       return protocol + "://" + address + ":" + port;
     }
+
+    Service(Security security) {
+      if (security.getSsl().isEnabled()) {
+        protocol = "https";
```

Comment:
Does this affect even single machine deployments? Intra-machine SSL on a single instance is not an efficient use of CPU